send-data/ingest.mdx

```diff
@@ -284,3 +284,13 @@ All events stored in Axiom must have a `_time` timestamp field. If the data you
 </Note>
 
 If you include the `_time` field in the ingested data, ensure the `_time` field contains timestamps in a valid time format. Axiom accepts many date strings and timestamps without knowing the format in advance, including Unix Epoch, RFC3339, or ISO 8601.
+
+## Best practices for ingesting data
+
+When ingesting data into Axiom, follow these best practices to optimize performance and reliability:
+
+- Batch events: Instead of sending individual events, group multiple events into a single request. This reduces the number of API calls and improves overall throughput.
+
+- Use Compression:  Enable gzip, zstd compression for your requests to reduce bandwidth usage and potentially improve response time.
+
+- Handle rate limiting and errors: Implement exponential backoff and retry logic to handle 429 (Too Many Requests) and 503 (Service Unavailable) errors. If ingestion consistently fails after retries, implement a fallback mechanism to store events locally or in cold storage for later processing.
```

Comment:
This could be intimidating for hobbyist or less-technical users. We should be clear that our libraries take care of this... assuming they do (and they should, if they don’t :-) )